src/modules/data/blocks/ticket/sagas.js

```diff
@@ -366,6 +366,15 @@ export function* setBodyDetails( clientId ) {
 		body.append( 'ticket[event_capacity]', yield select( selectors.getTicketsTempSharedCapacity ) );
 	}
 
+	const showSalePrice = yield select( selectors.showSalePrice, props );
+
+	if ( showSalePrice ) {
+		body.append( 'ticket[sale_price][checked]', yield select( selectors.getTempSalePriceChecked, props ) );
+		body.append( 'ticket[sale_price][price]', yield select( selectors.getTempSalePrice, props ) );
+		body.append( 'ticket[sale_price][start_date]', yield select( selectors.getTicketTempSaleStartDate, props ) );
+		body.append( 'ticket[sale_price][end_date]', yield select( selectors.getTicketTempSaleEndDate, props ) );
```

Comment:
To avoid the `max-len` eslint error:
```suggestion
		body.append(
			'ticket[sale_price][checked]',
			yield select( selectors.getTempSalePriceChecked, props )
		);
		body.append(
			'ticket[sale_price][price]',
			yield select( selectors.getTempSalePrice, props )
		);
		body.append(
			'ticket[sale_price][start_date]',
			yield select( selectors.getTicketTempSaleStartDate, props )
		);
		body.append(
			'ticket[sale_price][end_date]',
			yield select( selectors.getTicketTempSaleEndDate, props )
		);
```